app/rbac/main.polar

```diff
@@ -8,19 +8,19 @@ allow(actor, action, resource) if
 # Can a user read a contract?
 has_permission(user: User, "read", contract: Contract) if
     contract matches Contract and
-    (user.level != "three" or
+    (user.level != "two" or
```

Comment:
Nit: storing these as string constants risks introducing bugs because of typos. Suggest defining constants that store these strings and then checking equality against those constants. DRY stuff.

Reply:
I will do as you suggest.